CoreVisualGraph/src/au/gov/asd/tac/constellation/graph/visual/plugins/blaze/AddCustomBlazePlugin.java

```diff
@@ -50,6 +52,7 @@
  */
 @ServiceProvider(service = Plugin.class)
 @NbBundle.Messages("AddCustomBlazePlugin=Add Custom Blazes")
+@PluginInfo(pluginType = PluginType.CREATE, tags = {"CREATE"})
```

Comment:
tag could possibly be MODIFY (similar to what you've done with RemoveBlazes). Else the RemoveBlazes tag should probably be DELETE

Reply:
Good point, I think modify works best here.